captum/insights/attr_vis/example.py

```diff
@@ -32,31 +29,32 @@ def get_classes() -> List[str]:
     return classes
 
 
-def get_pretrained_model() -> Net:
-    class Net(nn.Module):
-        def __init__(self) -> None:
-            super(Net, self).__init__()
-            self.conv1 = nn.Conv2d(3, 6, 5)
-            self.pool1 = nn.MaxPool2d(2, 2)
-            self.pool2 = nn.MaxPool2d(2, 2)
-            self.conv2 = nn.Conv2d(6, 16, 5)
-            self.fc1 = nn.Linear(16 * 5 * 5, 120)
-            self.fc2 = nn.Linear(120, 84)
-            self.fc3 = nn.Linear(84, 10)
-            self.relu1 = nn.ReLU()
-            self.relu2 = nn.ReLU()
-            self.relu3 = nn.ReLU()
-            self.relu4 = nn.ReLU()
-
-        def forward(self, x):
-            x = self.pool1(self.relu1(self.conv1(x)))
-            x = self.pool2(self.relu2(self.conv2(x)))
-            x = x.view(-1, 16 * 5 * 5)
-            x = self.relu3(self.fc1(x))
-            x = self.relu4(self.fc2(x))
-            x = self.fc3(x)
-            return x
+class Net(nn.Module):
+    def __init__(self) -> None:
+        super(Net, self).__init__()
+        self.conv1 = nn.Conv2d(3, 6, 5)
+        self.pool1 = nn.MaxPool2d(2, 2)
+        self.pool2 = nn.MaxPool2d(2, 2)
+        self.conv2 = nn.Conv2d(6, 16, 5)
+        self.fc1 = nn.Linear(16 * 5 * 5, 120)
+        self.fc2 = nn.Linear(120, 84)
+        self.fc3 = nn.Linear(84, 10)
+        self.relu1 = nn.ReLU()
+        self.relu2 = nn.ReLU()
+        self.relu3 = nn.ReLU()
+        self.relu4 = nn.ReLU()
+
+    def forward(self, x):
```

Comment:
Please add typing to the forward function.
```
def forward(self, x:Tensor)->Tensor:
```

Reply:
I've added typing to the forward function.